server/src/@common/filters/all-exceptions.filter.ts

```diff
@@ -30,7 +31,7 @@ export class AllExceptionsFilter implements ExceptionFilter {
             e
         );
 
-        if (!(e instanceof UnauthorizedException) && !(e instanceof NotFoundException)) {
+        if (e instanceof InternalServerErrorException) {
```

Comment:
Has this been tested that it catches all errors appart from 401 and 404?

Reply:
We don't want it catching any of the 400s. Some other 500s we should probably get, I'll come back to this